nodes/containers/nodes-containers-using.adoc

```diff
@@ -40,7 +39,39 @@ To avoid losing containers due to kernel memory issues, ensure that the containe
 $(nproc) X 1/2 MiB
 ----
 
+[id="nodes-containers-runtimes"]
+== About container runtimes
 
+The container runtime runs and manages the components required to deploy and operate containers. The container runtime, also called the lower-level container runtime, is used by the CRI-O container engine to perform such tasks as:
```

Comment:
I think it would be useful (somewhere) to make clear how some folks use `container runtime` to reference the piece of the stack cri-o operates, whereas we try to make clear cri-o is a `container manager`. For instance, in lots of the kubernetes docs, they reference  `runtime` to refer to cri-o or containerd